### PR TITLE
plugin: feature: git: cloc: Log if no binaries are in path

### PR DESCRIPTION
**Pull request by sudharsana-kjl**
### Added

- Added LOGGER.error if a binary is not in path

### CODE REVIEW

**Comment by pdxjohnny** [comment]
Awesome! Last thing, can you please add this to the changelog too.

**Comment by sudharsana-kjl** [comment]
Is it fine if I specify something like this?

### Added
- Git feature to count lines of code

**Comment by pdxjohnny** [comment]
Exactly! You're PR descriptions are exactly what should go under unreleased in CHANGELOG.md you've been doing great work so make sure to add it to the changelog so that people will know what you did for the next release! 🤘

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - New Feature creation tutorial
 - New Model creation tutorial
 - Added update functionality to the CSV source
+- Git feature to count lines of code
```
Comment (pdxjohnny) [inline CHANGELOG.md]: The description of your PR is what should be added (although this is more of a change than an addition), aka

```
### Changed
- Git feature cloc logs if no binaries are in path
```

Comment (sudharsana-kjl) [reply CHANGELOG.md]: Thanks! I've made the changes in the next commit. Please check.

**Pull request merged**